ci/run_benchmark_pytests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+# Support invoking run_pytests.sh outside the script directory
+cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../
```

Comment:
For the sake of common patterns across each of these, I see the reason why this is here. However, it is a little unintuitive to me how we immediately pushd away from this folder, and rely on PATH/PYTHONPATH for choosing code to run. Is it worth removing this first cd here, or is that just nitpicking not worth the time?

Reply:
The pushd isn't strictly necessary in environments that support `PYTHONSAFEPATH=1`, so I figured it'd eventually get removed.